readme.md

```diff
@@ -46,14 +44,20 @@ $ npm init -y
 $ npm install --save express
 ```
 
-Q. What do you think npm is? (ST-WG)
 ---
 
+<summary>
+Q. What did npm do just now? (ST-WG)
+
+<details>
 > npm, short for node package manager. Allows us to install dependencies for our
-nodeJS application. Much like how gems did this for us in Rails.
+nodeJS application. Much like how the bundler did this for gems in Rails.
 
 `$ npm init` will initialize a new NodeJS application. Upon initialization it
 will prompt you for some user input to update the package.json. Using the `-y` argument allows you to use the defaults and not prompt for any options.
+</details>
+</summary>
```

Comment:
`<summary>` should be wrapped inside `<details>`

I'm guessing `Q. What did npm do just now?` should go inside `<summary>`